### PR TITLE
Feature/s3 empty bucket name

### PR DESCRIPTION
**Pull request by chennan47**
<b>Purpose</b>
Add warning for creating S3 bucket with an empty bucket name. Closes https://github.com/CenterForOpenScience/osf.io/issues/3957

<b>Changes</b>
Before Change:
There is no warning or anything happen when you click the submit button with no bucket name.
After Change:
screen shot 2015-08-06 at 12 13 02 pm

### CODE REVIEW

File: website/addons/s3/static/s3NodeConfig.js [hunk 1]
```diff
@@ -317,6 +317,9 @@ ViewModel.prototype.openCreateBucket = function() {
                                 '<label class="col-md-4 control-label" for="bucketName">Bucket Name</label> ' +
                                 '<div class="col-md-8"> ' +
                                     '<input id="bucketName" name="bucketName" type="text" placeholder="Enter bucket name" class="form-control" autofocus> ' +
+                                    '<div>' +
+                                        '<span id="errorMessage" ></span>' +
```
Comment (sloria) [inline website/addons/s3/static/s3NodeConfig.js]: `errorMessage` is too generic; it's too likely that the same ID might be used elsewhere on the page. Can you make it more specific, like `bucketModalErrorMessage`?

**Comment by sloria** [comment]
Will the error message be removed if the modal is dismissed then shown again?

**Comment by sloria** [comment]
Can we use the HTML `required` attribute here?

**Comment by chennan47** [comment]
@sloria since it is a bootbox modal, the required attribute doesn't work here. And even though it works, the feedback user received won't be consistent with other addon. The error message will be removed when the modal is dismissed.

File: website/addons/s3/static/s3NodeConfig.js (hunk 1 above)
Comment (chennan47) [inline website/addons/s3/static/s3NodeConfig.js]: Addressed in 98d2cca

**Pull request merged**